fedimint-core/src/config.rs

```diff
@@ -190,6 +190,20 @@ impl ClientConfig {
     pub fn federation_id(&self) -> FederationId {
         self.global.federation_id()
     }
+
+    /// Get the value of a given meta field
+    pub fn meta<V: serde::de::DeserializeOwned>(
+        &self,
+        key: &str,
+    ) -> Result<Option<V>, crate::encoding::DecodeError> {
```

Comment:
Why are you using `DecodeError` here? This isn't doing any binary encoding, where this error belongs to. By now it's an anyhow error anyway internally, so might as well just go with that.

Reply:
ah yes. fixed!